RecoLocalTracker/SiPixelRecHits/src/PixelCPEClusterRepair.cc

```diff
@@ -80,8 +80,7 @@ PixelCPEClusterRepair::PixelCPEClusterRepair(edm::ParameterSet const& conf,
   speed_ = conf.getParameter<int>("speed");
   LogDebug("PixelCPEClusterRepair::PixelCPEClusterRepair:") << "Template speed = " << speed_ << "\n";
 
-  GlobalPoint center(0.0, 0.0, 0.0);
-  float theMagField = mag->inTesla(center).mag();
+  float theMagField = mag->nominalValue();
```

Comment:
@perrotta, apparently this:
```
theMagField = mag->nominalValue();
```
returns the nominal magnetic field in kGauss and not in Tesla... I'll fix it in the next commit